commit 4c1f1371e98c387cb07043c26b99beff5b5111a8
Author: Alan Wright <alan@spotify.com>
Date:   Mon Apr 28 13:53:48 2014 +0100

    Throw exception in fixture destructor instead of FAIL().

diff --git a/src/test/util/LuceneTestFixture.cpp b/src/test/util/LuceneTestFixture.cpp
index 27c61e0..e737bdb 100644
--- a/src/test/util/LuceneTestFixture.cpp
+++ b/src/test/util/LuceneTestFixture.cpp
@@ -21,7 +21,7 @@ LuceneTestFixture::~LuceneTestFixture() {
     if (ConcurrentMergeScheduler::anyUnhandledExceptions()) {
         // Clear the failure so that we don't just keep failing subsequent test cases
         ConcurrentMergeScheduler::clearUnhandledExceptions();
-        FAIL() << "ConcurrentMergeScheduler hit unhandled exceptions";
+        boost::throw_exception(RuntimeException(L"ConcurrentMergeScheduler hit unhandled exceptions"));
     }
 }
 
